src/design-tokens/theme.ts

```diff
@@ -84,7 +84,7 @@ export const getTheme = (themeMode: ThemeMode) => {
   const palette = themeModes[themeMode];
   return createMuiTheme({
     typography: {
-      fontFamily: 'inherit',
+      fontFamily: ['-apple-system', 'BlinkMacSystemFont', '"Helvetica Neue"', 'Arial', 'sans-serif'].join(','),
```

Comment:
do we really want to change our font-family? I kinda liked Roboto 😁

Reply:
Yeah, I wana keep it but it brings a lot of headaches for people using proxies. The main issue is the `relative path` is generated in build time by webpack eg: `url(/static/xxx.xx)` this is a big issue if a reverse proxy has a relative path instead root path. I've tried to fix it last year but did not work out, I have a piple of duplicated issues that report that. So my solution is get rid of the assets, we use Emotion and the `<styles>` are being generated with JS, images and svg are base64 by default, so the Fonts were the next barrier, actually looks the same, I think we can precint of them. 

There is another thing I need to do in a major release, is a single JS server with the UI emebebed, to achieve this also you cannot have fonts, so, they will be gone somehow eventually anyway. 

My next step, is restore what I broke on `v4.1.2` on the next minor and try to close all those tickets.

Reply:
Oh, I understand. Yes, if removing the fonts helps to resolve the issue, then it is better to remove them, and we can maybe try to find another solution soon. Thanks for the great explanation!

Reply:
I'll keep you posted :)